app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    sum(split_integer(6, 2)) == 6
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    set(split_integer(6, 2)) == {3}
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    split_integer(6, 1) == [6]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    split_integer(32, 6) == [5, 5, 5, 5, 6, 6]
 
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+    split_integer(1, 2) == [0, 1]
```

Comment:
And here as well, you need to use `assert` to perform the test: `assert split_integer(1, 2) == [0, 1]`.